src/components/Speciality/Community.js

```diff
@@ -26,18 +26,28 @@ const ImgContainerCol = styled(Col)`
   justify-content: center;
 `
 
-const CommunitySection = ({ logo, text, background, title }) =>
-  text ? (
+const CommunitySection = ({
+  communityText,
+  communityBackgroundTitle,
+  communityBackgroundUrl,
+  communityLogoTitle,
+  communityLogoUrl,
+  title
+}) =>
+  communityText ? (
     <BlueBackground>
       <Trimmer>
-        <TrimmedImage alt={background.title} src={background.file.url} />
+        <TrimmedImage
+          alt={communityBackgroundTitle}
+          src={communityBackgroundUrl}
+        />
         <Padding top={5} bottom={5}>
           <Row>
             <ImgContainerCol width={[1, 1, 1, 1, 6 / 12]}>
-              {logo ? (
+              {communityLogoTitle ? (
                 <img
```

Comment:
I'm surprised we don't use the Common/Image.js component here as it accepts an image object with title and file keys (file also having url as a key).
Using ing works fine if we pass the values of `alt` and `src` directly but I don't see a change in the data being passed. Is this coming from GraphQL somewhere.

Reply:
@GBouffard where you're wondering about the formation of the title and url I believe this is coming from the transformation in `speciality.js`

On a whole Guilluame is right, we should be using the `Image` component from `Common/` but we should also be using the `GatsbyContentfulFluid_withWebp` format of the image itself not just a straight up file.

Reply:
And also `RatioContainer`

Reply:
Good point, done!